components/Result/Result.scss

```diff
@@ -0,0 +1,99 @@
+.result {
+  display: none;
+
+  &__wrapper {
+    @include tablet {
+      display: flex;
+      align-items: center;
+      gap: 24px;
+    }
+
+    @include desktop-xl {
+      gap: 32px;
+    }
+  }
+
+  &__cards {
+    @include tablet {
+      margin-left: auto;
+      width: calc(50% - 12px);
+
+      > *:not(:last-child) {
+        margin-bottom: 24px;
+      }
```

Comment:
here it is better to use & + & with margin-top: 24px. The logic will work like this, everything except the first one will be indented from the top. [Link](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors/Combinators) to a detailed explanation of the work